Makefile

```diff
@@ -1,73 +1,29 @@
 PIP := python -m pip --disable-pip-version-check
 WEBPACK := yarn build-acceptance
 
-bootstrap: develop init-config run-dependent-services create-db apply-migrations build-platform-assets
-
-develop:
-	@./scripts/do.sh develop
-
-clean:
-	@echo "--> Cleaning static cache"
-	rm -rf dist/* static/dist/*
-	@echo "--> Cleaning integration docs cache"
-	rm -rf src/sentry/integration-docs
-	@echo "--> Cleaning pyc files"
-	find . -name "*.pyc" -delete
-	@echo "--> Cleaning python build artifacts"
-	rm -rf build/ dist/ src/sentry/assets.json
-	@echo ""
-
-init-config:
-	@./scripts/do.sh init-config
-
-run-dependent-services:
-	@./scripts/do.sh run-dependent-services
-
-DROPDB := $(shell command -v dropdb 2> /dev/null)
-ifndef DROPDB
-	DROPDB = docker exec sentry_postgres dropdb
-endif
-CREATEDB := $(shell command -v createdb 2> /dev/null)
-ifndef CREATEDB
-	CREATEDB = docker exec sentry_postgres createdb
-endif
-
-drop-db:
-	@echo "--> Dropping existing 'sentry' database"
-	$(DROPDB) -h 127.0.0.1 -U postgres sentry || true
-
-create-db:
-	@echo "--> Creating 'sentry' database"
-	$(CREATEDB) -h 127.0.0.1 -U postgres -E utf-8 sentry || true
-
-apply-migrations:
-	@./scripts/do.sh apply-migrations
-
-reset-db: drop-db create-db apply-migrations
+bootstrap \
+develop \
+clean \
+init-config \
+run-dependent-services \
+drop-db \
+create-db \
+apply-migrations \
+reset-db \
+setup-git \
+node-version-check \
+install-js-dev \
+install-py-dev :
+	@./scripts/do.sh $@
+
+build-platform-assets \
```

Comment:
These 3 targets are executed in the CI or direnv when there's not a loaded venv, thus, using `SENTRY_NO_VENV_CHECK=1`

Reply:
upgrade-pip should be in the venv, but you should just remove it. It's just used by install-py-dev.

Reply:
Oh, well, it's used by getsentry actually, so just move it to normal do.sh.

Reply:
Unfortunately, it's also used as part of the CI outside of a venv, thus, need to set `SENTRY_NO_VENV_CHECK`.

Reply:
Oh.

Reply:
COuld be something worth improving, seems kinda fragile.